dfmux/include/dfmux/DfMuxCollector.h

```diff
@@ -50,6 +53,10 @@ class DfMuxCollector {
 	int Start();	// Start listening thread
 	int Stop();	// Stop listening thread
 
+	// Configure iceboard clock rate
+	void SetClockRate(double rate);
+	double GetClockRate() { return clock_rate_; };
```

Comment:
Does this get initialized by default to 100 MHz anywhere for old DfMux?

Reply:
Yep, in the constructors.

Reply:
Ah, indeed. Could you add a comment about the default somewhere?

Reply:
Added a comment in the source code and a note in the python bindings for `clock_rate`.